wiki/Tournaments/RMoT_Invitational_#2/en.md

```diff
@@ -0,0 +1,75 @@
+| Event                               | Date        |
+|------------------------------------:|:------------|
+| Round of 16                         | 6 May 2017  |
+| Quarterfinals and Losers Round 1    | 7 May 2017  |
+| Semifinals and Losers Round 2 and 3 | 13 May 2017 |
+| Losers Round 4, 5 and 6             | 14 May 2017 |
+| Finals and Grand Finals             | 14 May 2017 |
+
+## Staff
+
+| Role              | Member                                                                                                                                                                                                                                                              |
+|-------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Host              | [![](/wiki/shared/flag/Si.gif)Redavor](https://osu.ppy.sh/u/redavor)                                                                                                                                                                                                |
+| Admins            | [![](/wiki/shared/flag/Us.gif)-You Watanabe-](), [![](/wiki/shared/flag/Si.gif)DomenBzo](), [![](/wiki/shared/flag/Au.gif)Grassy](), [![](/wiki/shared/flag/Si.gif)kristi71111](), [![](/wiki/shared/flag/Tr.gif)oralekin]()                                        |
+| Referees          | [![](/wiki/shared/flag/Ph.gif)Deadpulse](), [![](/wiki/shared/flag/Ph.gif)kristoffer09](), [![](/wiki/shared/flag/Sg.gif)Kumabread](), [![](/wiki/shared/flag/Pl.gif)MagBlood](), [![](/wiki/shared/flag/Ph.gif)Misery](), [![](/wiki/shared/flag/Gb.gif)Stedoss]() |
+| Mappool Selectors | [![](/wiki/shared/flag/Th.gif)GuChangeDeWorld](), [![](/wiki/shared/flag/Ca.gif)Meow-Mix]()                                                                                                                                                                         |
+| Streamers         | [![](/wiki/shared/flag/Se.gif)BeatingHeart](), [![](/wiki/shared/flag/Pl.gif)FreezingTooEasy]()                                                                                                                                                                     |
+| Commentators      | [![](/wiki/shared/flag/Pl.gif)bsurma](), [![](/wiki/shared/flag/Se.gif)ErroTheCube](), [![](/wiki/shared/flag/Ph.gif)kristoffer09](), [![](/wiki/shared/flag/De.gif)Niegth](), [![](/wiki/shared/flag/Si.gif)ultraregen]()                                          |
+
+## Prizes
+
+The Prizes are handed out in the form of Points. 36 points can be redeemed for a month of osu! Supporter. Invitational winners are awarded twice the amount awarded in RMoT's.
+The winner of the Invitational tournament gets 96 points, 2<sup>nd</sup> place gets 48 points etc..
```

Comment:
@smoogipooo we have our own "point" system for redeeming supporter, we can change the name to Rapid points or something if it's a problem.